doc/user/content/overview/architecture.md

```diff
@@ -8,23 +8,124 @@ menu:
     weight: 10
 ---
 
-Materialize is a horizontally scalable, highly available distributed system. It separates storage and compute into different layers, allowing data ingestion, computation, and query serving to be **performed and scaled independently**.
+Materialize is a streaming database designed to provide correct, low latency answers to queries that need to be answered repeatedly by **incrementally maintaining answers** to those queries. Materialize enables users to ingest as much data as they want (by separating compute and storage into decoupled layers), perform as many queries as they want (by being horizontally and vertically scalable), while being highly available and flexible to operate (by offering active-active replication).
+
+
+The Materialize architecture is broken into three separate layers:
+* Storage: which is responsible for reading from and writing to external sources and sinks.
+* Compute: which is responsible for incrementally maintaining the results of user requested queries.
+* SQL Control Plane (Adapter): which is responsible for actually responding to user requests, and serving results back to users.
+
+Each of these components perform their work independently and can be **scaled independently**.
+
+TODO: give an overview of what this doc will talk about and also not talk about.
 
 {{<
     figure src="/images/architecture_overview.jpg"
     alt="Materialize architecture overview"
     width="700"
 >}}
 
-## Logical components
+## Theory background
```

Comment:
This section is too in the weeds. I'd jump straight into the layers you mentioned above.

Reply:
done.